cluster/image.go

```diff
@@ -21,18 +21,31 @@ func (image *Image) Match(IDOrName string, matchTag bool) bool {
 		return true
 	}
 
+	repo := ""
 	name := IDOrName
+	if strings.Contains(IDOrName, "/") {
+		parts := strings.SplitN(IDOrName, "/", 2)
+		repo = parts[0] + "/"
+		name = parts[1]
+	}
```

Comment:
This code is misleading and hard to read (I'm talking about this function overall after the fix). My suggestions are the following:

1. Don't put '/' to `repo` as it is not a part of repository name!
1. Finish parsing here - add `tag` var and parse `parts[1]` into real `name` and `tag` instead of this "maybe" `name`
1. We clearly have copy-paste here and 10 lines down, so move this parsing to a separate function!
1. It would be great if this thing could be synced with Docker Daemon code. (I couldn't find it there)